src/fides/api/ops/service/connectors/saas_connector.py

```diff
@@ -410,27 +415,62 @@ def run_consent_request(
         policy: Policy,
         privacy_request: PrivacyRequest,
         identity_data: Dict[str, Any],
-        executable_preferences: List[Consent],
+        consent_preferences: List[Consent],
```

Comment:
Locally, set only one consent option to executable is True to properly test this.  Another PR is updating how the customer configures whether a consent option is executable.